source/fundamentals/linq.txt

```diff
@@ -587,6 +588,169 @@ in the Atlas manual. For more examples about running Atlas Vector Search queries
 {+driver-short+}, see :atlas:`Run Vector Search Queries </atlas-vector-search/vector-search-stage/>`
 in the Atlas manual and select :guilabel:`C#` from the language dropdown.
 
+Bitwise Operators
+~~~~~~~~~~~~~~~~~
+
+This section describes the :wikipedia:`bitwise operators <Bitwise_operation>`
+supported by the {+driver-short+} that you can use in an aggregation pipeline.
+You can use multiple bitwise operators in the same 
+stage. The following guidelines apply when using bitwise operators:
+
+- All operands must be of type ``int`` or ``long``.
+
+- ``$bitAnd``, ``$bitOr``, and ``$bitXor`` take two or more operands. ``$bitNot`` takes one operand.
+
+- Bitwise operations are evaluated from left to right.
+
+The examples in this section use the following documents in a collection called
+``ingredients``:
+
+.. code-block:: json
+
+   { "_id": 1, "name": "watermelon", "price": 5, "count": 1 },
+   { "_id": 2, "name": "onions", "price": 2, "count": 4 },
+   { "_id": 3, "name": "eggs", "price": 5, "count": 12 },
+   { "_id": 4, "name": "potatoes", "price": 3, "count": 0 },
+   { "_id": 5, "name": "pasta", "price": 4, "count": 100 },
+   { "_id": 6, "name": "cheese", "price": 4 }
```

Comment:
This final document can't have been created using the `Ingredient` POCO because `Count` is an `int` and can't be `null`.

Also, the element names in these sample documents don't match the field names in the POCO (the C# properties start with a capital letter).

Reply:
The data types used in the example have been changed to be nullable.

As for the second point, for these examples we use a convention pack (see the bottom of the [Overview section](https://www.mongodb.com/docs/drivers/csharp/current/fundamentals/linq/) on this page) to convert snake case field names into Pascal case.

Reply:
It's hard to tell from looking at just this PR that there is a convention somewhere else that is mapping the property names. That's confusing.